packages/core/js-client/src/particle/Particle.ts

```diff
@@ -47,12 +52,27 @@ export class Particle implements IParticle {
             toUint8Array(json.data),
             json.ttl,
             json.init_peer_id,
+            toUint8Array(json.signature)
         );
 
         return res;
     }
 }
 
+/**
+ * Builds particle message for signing
+ */
+export const buildParticleMessage = ({ id, timestamp, ttl, script }: Omit<IParticle, 'initPeerId' | 'signature' | 'data'>): Uint8Array => {
```

Comment:
intead of using the word `message` here and in the places where this function is used, I would suggest using probably a bit more verbose but I believe a better name `particleDataToBeVerified`

so this function could be `buildParticleDataToBeVerified`

Reply:
The terminology suggest that you sign a message, not data. And signature is a effectively signed **message**. 

_A signing algorithm that, given a message and a private key, produces a signature._

Reply:
`verify` method's typescript signature calls it `data` though. Don't know where you'v seen that `message` quote. Maybe it's valid as well but with crypto (from what I am familliar with) it's always called data. We enrypt data, verify data, sign some data and so on. I just never encountered the word `message` in this context, that's why it sounded a bit weird to me when I first saw it. I am not really pushing hard for the renaming, just curious if I am alone in this, or somebody else has the same impression

Reply:
Message terminology came from wikipedia https://en.wikipedia.org/wiki/Digital_signature

Reply:
That's just my crypto illiteracy. Feel free to leave it as it is. Thanks for response